frontend/app/src/components/UserDropdown.vue

```diff
@@ -106,3 +161,129 @@ export default class UserDropdown extends Vue {
   }
 }
 </script>
+
+<style scoped lang="scss">
+/* stylelint-disable */
```

Comment:
Disabling a few specific rules case by case is ok but I would advise against disabling all the style rules.

Reply:
I already move the disabled comment to only specific rule